From 8b973e939c1c41bf85504d80e3c9cf193893d5e7 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Mon, 27 Nov 2023 10:35:53 +0000
Subject: [PATCH 17/21] Use PYTHON for build wheel

Tensorflow tries to use a Python binary it downloaded as part of its
hermetic Python approach. As we use our own Python toolchain, this is
not existing. Instead, point it to our Python installation.
---
 tensorflow/tools/pip_package/build_pip_package.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tensorflow/tools/pip_package/build_pip_package.sh b/tensorflow/tools/pip_package/build_pip_package.sh
index ff21aadba95..637faec06cb 100755
--- a/tensorflow/tools/pip_package/build_pip_package.sh
+++ b/tensorflow/tools/pip_package/build_pip_package.sh
@@ -352,7 +352,7 @@ function build_wheel() {
 
   rm -f MANIFEST
   echo $(date) : "=== Building wheel"
-  $FULL_DIR setup.py bdist_wheel ${PKG_NAME_FLAG} >/dev/null
+  $PYTHON setup.py bdist_wheel ${PKG_NAME_FLAG} >/dev/null
   mkdir -p ${DEST}
   cp dist/* ${DEST}
   popd > /dev/null
